Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn app:app_routes
```

Comment:
web: gunicorn run:app

then in the root folder
run.py
` from app.app_routes import app`
`app.run(debug=False)`
If Heroku deployment still fails, please check if the python version 3.6.7 is supported on Heroku